src/genie/libs/parser/junos/show_security.py

```diff
@@ -0,0 +1,112 @@
+"""show_security.py
+JunOS parsers for the following show commands:
+    * show security policies hit-count
+"""
+# Python
+import re
+
+# Metaparser
+from genie.metaparser import MetaParser
+from pyats.utils.exceptions import SchemaError
+from genie.metaparser.util.schemaengine import (Any, Optional, Use, Schema, ListOf)
+
+
+class ShowSecurityPoliciesHitCountSchema(MetaParser):
+    """ Schema for:
+            * show security policies hit-count
+    """
+    """schema = {
+        "security_policy_counts": {
+            str: {
+                "security_policy": ListOf(
+                    {
+                        "index": str,
+                        "from_zone": str,
+                        "to_zone": str,
+                        "name": str,
+                        "policy_hit_count": str
+                    },
+                )
+                "total_policies": str
+            }
+        }
+    }"""
+
+    # Main Schema
+    schema = {
+        "security_policy_counts": {
+            str: {
+                "security_policy": ListOf(
+                    {
+                        "index": str,
+                        "from_zone": str,
+                        "to_zone": str,
+                        "name": str,
+                        "policy_hit_count": str,
+                    }
+                ),
+                "total_policies": str
+            }
+        }
+    }
+
+
+class ShowSecurityPoliciesHitCount(ShowSecurityPoliciesHitCountSchema):
+    """ Parser for:
+            * show security policies hit-count
+    """
+    cli_command = 'show security policies hit-count'
+
+    def cli(self, output=None):
+
+        if not output:
+            out = self.device.execute(self.cli_command)
+        else:
+            out = output
+
+        ret_dict = {}
+
+        # Logical system: root-logical-system
+        p1 = re.compile(r'^(Logical system:)\s(?P<logical_system>\S+)$')
+
+        # 9       junos-global     junos-global      GLOBAL-PERMIT-SSH-IN 541543
+        # 10      junos-global     junos-global      GLOBAL-PERMIT-KNOWN 337917
+        # 11      untrust          UNTRUST-STRICT        STRICT-PERMIT-NTP-IN 1243
+        # 12      untrust          UNTRUST-STRICT        STRICT-PERMIT-SSH-IN 445265
```

Comment:
Examples 9 and 11 should be enough to get the idea across, 10 and 12 are unnecessary.